src/main/java/com/toptrumps/db/Statistics.java

```diff
@@ -0,0 +1,44 @@
+package com.toptrumps.db;
+import java.sql.*; 
+import java.sql.Connection; 
+import java.sql.ResultSet; 
+import java.sql.SQLException; 
+import java.sql.Statement; 
+
+public class Statistics{
+
+    private int games_played;
+    private int ai_wins;
+    private int human_wins;
+    private int draws_average;
+    private int max_rounds; 
```

Comment:
this_is_not_python => cammelCasePlease

Reply:
Sorry! Was making it 1:1 with the DB relation and got distracted.